apis/logging/v1/output_types.go

```diff
@@ -294,6 +289,11 @@ type Splunk struct {
 	// If IndexKey && IndexName are not specified, the default index defined within Splunk is used.
 	// +optional
 	IndexName string `json:"indexName,omitempty"`
+
+	// Deprecated. Fields to be added to Splunk index. Use indexKey or indexName instead.
```

Comment:
I don't think there is a relation between "Fields" and index key/name. The former is meta that is indexed where the latter provides the "root" name to store the data.  I would drop the last sentance